containers/jetty/src/main/java/com/yahoo/athenz/container/AthenzJettyContainer.java

```diff
@@ -469,6 +470,23 @@ void addHTTPSConnector(HttpConfiguration httpConfig, int httpsPort, boolean prox
             sslConnector.addBean(connectionLogger);
         }
         server.addConnector(sslConnector);
+
+        // Reload the key-store if the file is changed
+System.err.println("=====GBENDOR:   Reload the key-store if the file is changed");
```

Comment:
please clean the error messages and use logger.

Reply:
you caught me in the middle of a "debugging" session...
;-)